src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSetsCompare/PermissionSetsCompareItem.js

```diff
@@ -0,0 +1,133 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import xor from 'lodash/xor';
+import PropTypes from 'prop-types';
+import { useTenantPermissions } from '../../../../../../../hooks';
+
+export function PermissionSetsCompareItem({
+  columnName,
+  permissionsToCompare,
+  selectedMemberOptions,
+  setPermissionsToCompare,
+}) {
+  const intl = useIntl();
+  const isMounted = useRef(false);
+  const [selectedMemberId, setSelectedMemberId] = useState('');
+  const [selectedPermissionId, setSelectedPermissionId] = useState('');
+  const showCallout = useShowCallout();
+
+  const handlePermissionsLoadingError = useCallback(({ response }) => {
+    const defaultMessage = intl.formatMessage({ id: 'ui-consortia-settings.errors.permissionSets.load.common' });
+
+    if (response?.status === 403) {
+      return showCallout({
+        message: `${defaultMessage} ${intl.formatMessage({ id: 'ui-consortia-settings.errors.permissionsRequired' })}`,
+        type: 'error',
+      });
+    }
+
+    return showCallout({
+      message: defaultMessage,
+      type: 'error',
+    });
+  }, [intl, showCallout]);
+
+  const {
+    isFetching,
+    permissions,
+  } = useTenantPermissions(
+    {
+      tenantId: selectedMemberId,
+      searchParams: {
+        query: 'mutable==true',
+        expandSubs: true,
+      },
+    },
+    { onError: handlePermissionsLoadingError },
+  );
+
+  const permissionOptions = useMemo(() => {
+    return permissions.map(({ id, displayName }) => ({ value: id, label: displayName }));
+  }, [permissions]);
+
+  const assignedPermissionsList = useMemo(() => {
+    const selectedPermission = permissions.find(({ id }) => id === selectedPermissionId);
+
+    return selectedPermission?.subPermissions?.map(({ displayName }) => displayName) || [];
+  }, [selectedPermissionId, permissions]);
+
+  const uniqueItems = xor(assignedPermissionsList, permissionsToCompare);
+  const itemFormatter = (item) => (<li key={item}>{uniqueItems.includes(item) ? <mark>{item}</mark> : item}</li>);
+  const isEmptyMessage = <FormattedMessage id="ui-consortia-settings.consortiumManager.members.permissionSets.compare.permissionSet.empty" />;
+
+  useEffect(() => {
+    if (isMounted.current) {
+      setPermissionsToCompare(assignedPermissionsList, columnName);
+    } else {
+      isMounted.current = true;
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [assignedPermissionsList]);
+
+  return (
+    <div>
+      <Selection
+        name="members"
+        label={<FormattedMessage id="ui-consortia-settings.consortiumManager.members.permissionSets.compare.member" />}
+        id="memberSelect"
+        placeholder={intl.formatMessage({ id: 'ui-consortia-settings.consortiumManager.members.permissionSets.compare.member.placeholder' })}
+        dataOptions={selectedMemberOptions}
+        onChange={(value) => setSelectedMemberId(value)}
+      />
+      <Selection
+        name="permission-set"
```

Comment:
What do you think, does it makes sense to disable selection if there is no selected member?